docs/DeveloperGuide.adoc

```diff
@@ -1775,34 +1775,76 @@ These instructions only provide a starting point for testers to work on; testers
 
 .. Download the jar file and copy into an empty folder
 .. Double-click the jar file +
-   Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   Expected: Shows the GUI. You should be greeted with a prompt to login or register. The window size may not be optimum.
 
 . Saving window preferences
 
 .. Resize the window to an optimum size. Move the window to a different location. Close the window.
 .. Re-launch the app by double-clicking the jar file. +
    Expected: The most recent window size and location is retained.
 
-_{ more test cases ... }_
+=== Registering a Student Account
+Below are some test cases you can perform to test the `RegisterCommand` for User Account Management feature.
+
+[NOTE]
+The below test cases only cater to `Student` accounts
+
+.   Test case: `register user/max33 pass/#Qwerty4321 n/Max Verstappen enroll/15/03/2015 maj/CS maj/DA min/BA min/MA` +
+    Expected: A new account is created. The `Student` details as provided in the command should be automatically displayed in `UserTab`. A successful message should be displayed in the Output Display. Additionally, a save location is provided to the user. Examine the project's root folder to determine if the file is indeed created.
+
+
+.   Test case: `register user/m@x33 pass/#Qwerty4321 n/Max Verstappen enroll/15/03/2015 maj/CS maj/DA min/BA min/MA` +
+    Expected: No new account is created. A invalid `Username` error should be shown in the the Output Display. Rest of the UI remains unchanged.
+
+.   Prerequisite: A Student Account with `Username` max33 already exists in `CredentialStore` +
+    Test case: `register user/max33 pass/#Qwerty4321 n/Max Verstappen enroll/15/03/2015 maj/CS maj/DA min/BA min/MA` +
+  Expected: No new account is created. Error details should be shown in the Output Display. Rest of the UI remains unchanged.
+
+. Other incorrect `register` commands to try: +
+.. `register user/max33 pass/Qwerty4321 n/Max Verstappen enroll/15/03/2015 maj/CS maj/DA min/BA min/MA` +
+.. `register user/max33 pass/#Qwerty4321 enroll/15/03/2015 maj/CS maj/DA min/BA min/MA`
+.. `register user/max33 pass/#Qwerty4321 n/Max Verstappen maj/CS maj/DA min/BA min/MA`
+.. `register user/max33 pass/#Qwerty4321 n/Max Verstappen enroll/15/03/2015 min/BA min/MA`
+
+
+=== Logging in into a Student Account
+Below are some test cases you can perform to test the `LoginCommand` for User Account Management feature.
+
+.   Prerequisite: A student account should already be registered(i.e. max33) and the respective userdata file be available. +
```

Comment:
It would be good to specify that `max33` is a `Username` since name can be non-unique, it might be a good idea to clarify it.